### DISCUSSION TITLE
Custom background image for page, and small edits to page structure?

### DISCUSSION THREAD

**Discussion by CursedBlackCat** [jpanther/lynx#75] [Q&A]
Hey there,

I was thinking of setting a background image with some blur just to make the page a bit more visually interesting/unique. Is there a way to set a custom background image for the entire page? Adding a `body` tag in `assets/css/custom.css` doesn't seem to work, and while adding `!important` does work, two further issues arise from that: firstly, that being bad practice/a kind of janky workaround that I'd like to avoid, and secondly, I can't seem to find where the relative path for the image would be (I've tried both `assets` and `static`).

Also, is there a way to make slight edits to the page structure, such as adding `<hr>` tags between "sections" of buttons/links or adding a small `<p>` block beneath the headline for a brief, 2-3 sentence bio?

Thanks in advance!

**Comment by emroch** [comment]
+1 for the background image and sectioning the links.
The bio is already possible. From the readme (hiding just below the custom link code block):
> Additional page content can be provided by creating a Markdown file at content/_index.md. The contents of this file will be displayed between the title and links. Check out the exampleSite to see this in practice.

**Reply by emroch** [reply]
I took a stab at implementing the sections and it works pretty well for my use case. See PR #94.
